### PR TITLE
WGB-184 Image zoom

### PR DESCRIPTION
**Pull request by ErnestsVerins**

### CODE REVIEW

File: src/app/component/ProductGallery/ProductGallery.container.js [hunk 1]
```diff
@@ -160,14 +174,14 @@ export class ProductGalleryContainer extends PureComponent {
             <Subscribe to={ [SharedTransitionContainer] }>
                 { ({ registerSharedElementDestination }) => (
                     <ProductGallery
-                      registerSharedElementDestination={ registerSharedElementDestination }
-                      { ...this.containerProps() }
-                      { ...this.containerFunctions }
+                        registerSharedElementDestination={ registerSharedElementDestination }
+                        { ...this.containerProps() }
+                        { ...this.containerFunctions }
```
Comment (yeegor) [inline src/app/component/ProductGallery/ProductGallery.container.js:179]: Wrong spacing here. JSX nesting considers 2 space indent, blocks - 4 space.

File: src/app/component/ProductGallery/ProductGallery.container.js [hunk 2]
```diff
@@ -160,14 +174,14 @@ export class ProductGalleryContainer extends PureComponent {
...
                     />
                 ) }
             </Subscribe>
         );
     }
 }
 
-export default ProductGalleryContainer;
+export default connect(null,mapDispatchToProps)(ProductGalleryContainer);
```
Comment (yeegor) [inline src/app/component/ProductGallery/ProductGallery.container.js:187]: Spacing after comma is required.

File: src/app/component/ProductGallery/ProductGallery.style.scss [hunk 3]
```diff
@@ -17,6 +17,138 @@
...
+            border: 2px solid grey;
+            opacity: 1;
+        }
+    }
+
+    &-Popup {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        & nav {
```
Comment (yeegor) [inline src/app/component/ProductGallery/ProductGallery.style.scss:97]: Consider adding paddings to blocks

File: src/app/component/Slider/Slider.component.js [hunk 4]
```diff
@@ -290,16 +300,16 @@ export default class Slider extends PureComponent {
...
-              block="Slider"
-              elem="Image"
-              mods={ { type: 'single' } }
-              // eslint-disable-next-line react/jsx-no-bind
-              onClick={ () => this.changeActiveImage(i) }
+                block="Slider"
+                elem="Image"
+                mods={ { type: 'single' } }
+                // eslint-disable-next-line react/jsx-no-bind
+                onClick={ () => this.changeActiveImage(i) }
```
Comment (yeegor) [inline src/app/component/Slider/Slider.component.js:307]: Wrong indentation in all the JSX.

File: src/public/assets/images/global/search.svg [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (yeegor) [inline src/public/assets/images/global/search.svg:1]: I believe that both search and arrow icons are implemented as CSS in the ScandiPWA core.

**Review by yeegor — CHANGES_REQUESTED** [review]
I went through the coding style and it needs improvement.

@alfredsgenkins I am not aware of what this actually is, I see the vertical scrollable gallery references from VGB, is this PR relevant?

**Comment by alfredsgenkins** [comment]
Closing, we took a different implementation. Thanks for contribution tho!

**Pull request closed without merging**